workflows/gasclustering.nf

```diff
@@ -71,11 +72,32 @@ workflow GASCLUSTERING {
     // Create a new channel of metadata from a sample sheet
     // NB: `input` corresponds to `params.input` and associated sample sheet schema
     input = Channel.fromSamplesheet("input")
-    merged_input = input.map{
-        meta, mlst_files -> mlst_files
+
+    merged_alleles = input.map{
+        meta, mlst_files,
+        metadata_1, metadata_2, metadata_3, metadata_4,
+        metadata_5, metadata_6, metadata_7, metadata_8 -> mlst_files
     }.collect()
 
-    merged = LOCIDEX_MERGE(merged_input)
+    metadata_headers = Channel.of(
+        tuple(
+            params.metadata_1_header, params.metadata_2_header,
+            params.metadata_3_header, params.metadata_4_header,
+            params.metadata_5_header, params.metadata_6_header,
+            params.metadata_7_header, params.metadata_8_header)
+        )
+
+    metadata_rows = input.map{
+        meta, mlst_files,
+        metadata_1, metadata_2, metadata_3, metadata_4,
+        metadata_5, metadata_6, metadata_7, metadata_8 -> tuple(
```

Comment:
If you use `"meta": ["metadata_1"]` (from other comment) in the `schema_input.json`, then the metadata values here will appear under `meta.metadata_1` for example.

Reply:
Yup, changed in a1f69e86065537799c16873a82bc4e013e7eb479 (as above).